content/tutorials/getting-started/javascript.md

```diff
@@ -139,7 +139,7 @@ const node = await Libp2p.create({
 
 Now that you have configured a **Transport**, **Crypto** and **Stream Multiplexer** module, you can start your libp2p node. We can start and stop libp2p using the [`libp2p.start()`](https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#start) and [`libp2p.stop()`](https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#stop) methods.
 
-A libp2p node needs to have a listen address for the given transport, so that it can be reached by other nodes in the network. Accordingly, we will install the `multiaddr` module to create a tcp [multiaddress](definition_muiltiaddress) and add it to the node.
+A libp2p node needs to have a listen address for the given transport, so that it can be reached by other nodes in the network. Accordingly, we will install the `multiaddr` module to create a tcp [multiaddress](https://docs.libp2p.io/reference/glossary/#multiaddr) and add it to the node.
```

Comment:
I believe the issue is below in this file. The definition of `definition_multiaddress` is linking into: https://docs.libp2p.io/tutorials/getting-started/javascript/definition_muiltiaddress but it should be starting from the origin path https://docs.libp2p.io/reference/glossary/#multiaddr

Reply:
Sorry, I don't think I'm following what you're suggesting. What do you mean by "it should be starting from the origin path ..."? Was the link that I used pointing into the glossary entry for the multiaddress not the intended link? Also, I guess I hadn't notice this before when I changed the link but it seems that `definition_muiltiaddress` was misspelled (there is an `i` between the `u` and `l`), but even after correcting the spelling, the link is still broken.

Reply:
I mean that there is a reference for it in the end of the document: https://github.com/libp2p/docs/pull/86/files#diff-2a666e46d0d9d65d1a1c33e40635e1b2R280

We should change it there. Ideally not using the entire url but a local path

Reply:
Ok, I see what you mean! Yeah the link to the glossary page should probably be a relative path instead of an absolute path to the page for stability. It seems that using references in markdown requires the format `[text][ref]` as opposed to `[text](ref)` which was causing the error. Hopefully, the fix I just did resolves that broken link issue.